### DISCUSSION TITLE
HTTPS only without using a proxy

### DISCUSSION THREAD

**Discussion by mahescho** [rustdesk/rustdesk#9276] [Q&A]
Is it possible to force a client to use HTTPS (TCP 443) only without using a proxy?

**Comment by rustdesk** [comment]
No, here is a tutorial of how to set up http proxy server, it is quite easy, https://github.com/rustdesk/rustdesk/wiki/Set-up-http-proxy-server

**Reply by mahescho** [reply]
In an corporate environment this is no problem to use a proxy. Even in home networks it isn't. Ye, I can open the required ports very selectively, but why? The question is: why is it possible to run every thing over 443 when a proxy is used but not when no proxy is used? I would expect that falling back to 443 only should be the default in case the other "normally" (443 only should be the default anyway) used ports are not available. What are the good reasons for not using 443 only?
